### ISSUE TITLE
pycharm2020 support

### ISSUE THREAD

**Issue by sobolevn** [jansorg/pycharm-flake8#3]
Hi! Thanks a lot for writting this plugin!

Currently some of my (https://github.com/wemake-services/wemake-python-styleguide) users report that it is impossible to install this plugin with the latest PyCharm version (2020.2).

Sorry that I don't have any other details.

**Comment by jansorg** [comment]
@sobolevn That's correct, the plugin is only available for earlier versions. The PyCharm team said, that they integrated the functionality of the plugin into the IDE itself. I've not tested that, but 2020.2 shouldn't need the plugin.
